sktime/transformations/series/multivariate_compositor.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+__all__ = ["MultivariateCompositor"]
+__author__ = ["Svea Meyer"]
+
+import pandas as pd
+from sklearn.base import clone
+from sktime.transformations.base import _SeriesToSeriesTransformer
+from sktime.utils.validation.series import check_series
+
+
+class MultivariateCompositor(_SeriesToSeriesTransformer):
+    """
+    Parameters
+    ----------
+    transformer : series-to-series transformer to be applied to each series
+    columns : names of columns that are supposed to be transformed
+    """
+
+    _required_parameters = ["transformer"]
+    _tags = {
+        "multivariate-only": True,
+    }
+
+    def __init__(self, transformer, columns="all"):
+        self.transformer = transformer
+        self.columns = columns
+        self.transformers_ = None
+        super(MultivariateCompositor, self).__init__()
+
+    def fit(self, Z, X=None):
+        """
+        Iterates over columns (series) and applies the fit function of the transformer
+        """
+        if not isinstance(Z, pd.DataFrame):
+            raise ValueError("Z needs to be a multivariate Pandas Series")
+        self._is_fitted = False
+        z = check_series(Z)
+        if self.columns == "all":
+            self.columns = z.columns
+
+        # make sure z contains all columns that the user wants to transform
+        Z_wanted_keys = set(self.columns)
+        Z_new_keys = set(z.columns)
+        difference = Z_wanted_keys.difference(Z_new_keys)
+        if len(difference) != 0:
+            raise ValueError("Missing columns" + str(difference) + "in Z.")
+
+        self.transformers_ = {}
+        for colname in self.columns:
+            transformer = clone(self.transformer)
+            self.transformers_[colname] = transformer
+            self.transformers_[colname].fit(z[colname], X)
+            # self.transformers_[colname].is_fitted = True
+        self._is_fitted = True
```

Comment:
```suggestion
```

Reply:
this is already contained in the parent class `BaseTransformer`

Reply:
ok it seems you overwrite `fit()` here, so in this case we need `self._is_fitted = True`. So this is also a task for refactoring, but not your task now I think :)

Reply:
hmm here I might be having a misconception, but my train of thought was that I would want to be able to check whether this overall transformer is fitted. I am aware that I am already setting is_fitted to true for all the single series transformers when calling fit() on them. But this then only contains information about the single series transformers being fitted and not whether fit() has been applied to this multivariate_compositor already.

Reply:
after refactoring `BaseTransformer`, we would have there a `_fit()` function that will contain `self._is_fitted = True`. But you can ignore my comment above, as I have seen that there is just a `fit()` function which is overwritten by each transformer currently. Didnt want to make confusion :)

Reply:
Ah, I did not see your second comment! yes, exactly I'm overwriting fit()